### DISCUSSION TITLE
How to clear sorting on a sorted table

### DISCUSSION THREAD

**Discussion by aranvir** [vincjo/datatables#118] [Q&A]
Hi, I'm just getting started with svelte and this datatables package and working my way through the tutorials (e.g. https://vincjo.fr/datatables/tutorial/sort-data).

It's not really clear to me how to "reverse" the sorting of a table besides doing a reload of the page. I did find the `DataHandler.clearSort` method, but it's not doing anything beside clearing the sorted property. This allows me to make the indicators look like the table is unsorted but in truth it just keeps the last sorting.

My guess is that the original sorting is just not stored so it can't be reversed. But I wanted to ask if that's truly the case and if there is a way to work around it (besides reloading the page). Thanks!

**Comment by aranvir** [comment]
Actually figured it out myself. Have data with a natural order field like `id` and then use a function like below to get the original order back.

```javascript
    function clearSort() {
        handler.sort("id")
        handler.clearSort();
    }
```

**Comment by vincjo** [comment]
indeed: `clearSort` remove the sort history (last 3 sort) but didn't get back to the original sort
